lib/service/customer_support_service.dart

```diff
@@ -498,4 +525,124 @@ class CustomerSupportServiceImpl extends CustomerSupportService {
       await createRenderingIssueReport(token, ["IPFS Loading"]);
     }
   }
+
+  @override
+  Future<List<Object>> getIssuesAndAnnouncement() async {
+    List<Object> result = [];
+    List<Issue> issues = await getIssues();
+    List<AnnouncementLocal> announcements =
+        await _announcementDao.getAnnouncements();
+
+    if (issues.isNotEmpty && announcements.isNotEmpty) {
+      for (var issue in issues) {
```

Comment:
I think you can simplify it like this:

`announcements.removeWhere((element) => issues.any((i) => i.announcementID == element.announcementContextId))`